src/uct/ib/dc/dc_mlx5.c

```diff
@@ -1256,6 +1239,24 @@ static void uct_dc_mlx5_iface_handle_failure(uct_ib_iface_t *ib_iface,
     }
 }
 
+static int
+uct_dc_mlx5_iface_is_reachable_v2(const uct_iface_h tl_iface,
+                                  const uct_iface_is_reachable_params_t *params)
+{
+    uct_dc_mlx5_iface_t *iface = ucs_derived_of(tl_iface, uct_dc_mlx5_iface_t);
+    uct_dc_mlx5_iface_addr_t *addr = (uct_dc_mlx5_iface_addr_t*)
+                                             params->iface_addr;
```

Comment:
check field_mask

Reply:
`uct_iface_is_reachable_v2()` is the only user of `->iface_is_reachable_v2()` functions, and is already checking `field_mask` for `device_addr` and `iface_addr`. Shall I:
1- keep it like this
2- move the checks inside each implementation
3- add checks in each implementation regardless

Reply:
fixed with 2-